econml/utilities.py

```diff
@@ -1482,3 +1482,29 @@ def jacify_featurizer(featurizer):
        a function for calculating the jacobian
     """
     return _TransformerWrapper(featurizer)
+
+
+def single_strata_from_discrete_arrays(arrs):
```

Comment:
[minor] The singular for strata is stratum, so this name seems slightly weird.  I think `strata_from_discrete_arrays` (since this gets the strata for all of the rows at once) is shorter and just as clear.